CMakeLists.txt

```diff
@@ -8,6 +8,9 @@ if(EXISTS ${LOC_PATH})
     message(FATAL_ERROR "You cannot build in the source directory (or any directory with a CMakeLists.txt file). Please make a build subdirectory.")
 endif()
 
+set(OR_GROUPSIZE 128)
+set(OR_GRAINSIZE 7)
```

Comment:
I would probably use the CMake.in, but either would work. I'd make them CMake options, too:

```cmake
set(GOOFIT_MPI_OR_GROUPSIZE 128 CACHE INT "The group size for goofit MPI")
...
```